package.json

```diff
@@ -24,11 +24,12 @@
     "url": "https://github.com/storybookjs/memoizerific/issues"
   },
   "type": "module",
-  "main": "memoizerific.js",
-  "module": "memoizerific.js",
+  "main": "dist/memoizerific.js",
+  "module": "dist/memoizerific.js",
```

Comment:
I do not think this is correct, we should output 2 module types, should be easy to do with ts-up!

Reply:
I updated it now, according to this guide https://antfu.me/posts/publish-esm-and-cjs